src/vulkan/compute_pipeline.cc

```diff
@@ -116,6 +116,14 @@ Result ComputePipeline::Compute(uint32_t x, uint32_t y, uint32_t z) {
       command_->GetCommandBuffer(), VK_PIPELINE_BIND_POINT_COMPUTE, pipeline);
   device_->GetPtrs()->vkCmdDispatch(command_->GetCommandBuffer(), x, y, z);
 
+  r = command_->End();
```

Comment:
This change looks unrelated? Good fix though.

Reply:
It's needed in order to finish the command and have the data available for probe. Without this we'd need the pipeline in order to submit the command correctly.